api/models/post.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const postSchema = Schema({
+  author: [
+    {
+      ref: "User",
+      type: Schema.Types.ObjectId,
+      required: true,
+    },
+  ],
+
+  date: {
+    type: String,
+    required: true,
+  },
+
+  title: {
+    type: String,
+    required: true,
+  },
+
+  content: {
+    type: String,
+    required: true,
+  },
+
+  likes: {
+    type: Number,
+  },
+
+  comments: [
+    {
+      type: Schema.Types.ObjectId,
+      ref: "Comment",
+    },
+  ],
+
+  tags: [
+    {
+      type: Schema.Types.ObjectId,
```

Comment:
Could be tag directly instead of id, since tag text is unique

Reply:
Fixed